webrecorder/browsertrix
# Update QA tab page list and standardize review statuses

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/1508

### Changes

- Paginates QA tab page list
- Sort QA tab page list by title, URL, comments, and approval; displays all sortable fields
- Standardizes page "reviews" -> "approvals"
- Standardizes crawl review rating labels
- Displays only latest comment in review page list

Main difference from mocks is I added a separate "Comments" column since we can sort by comment count.

### Manual testing

1. Log in as crawler
2. Go to Archived Items and click archived item with review
3. Click QA. Verify "Crawl Rating" and page list renders as expected
4. Update page list sorting and pagination controls. Verify list updates as expected
5. Click page in list. Verify you've navigated to page in crawl review view

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Archived Item - QA tab | <img width="1044" alt="Screenshot 2024-04-08 at 4 13 54 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/ae0d688f-d0a7-4646-a6ef-b4bde8e550fa"> |
| Crawl Review - Pages list sort | <img width="328" alt="Screenshot 2024-04-08 at 4 15 24 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/3f14b8b0-0a0e-4fe9-bbae-1def503fc1fd"> |
| Crawl Review - Pages list filter | <img width="582" alt="Screenshot 2024-04-08 at 12 54 20 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/b634d34f-85d5-4670-8faf-8f3a60ae11a8"> |
